semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtils.java

```diff
@@ -118,4 +131,88 @@ public static boolean isRpcTypeGrpcForOTelFormat(Map<String, AttributeValue> val
     }
     return Optional.empty();
   }
+
+  public static int getGrpcStatusCode(Event event) {
+    Map<String, AttributeValue> attributeValueMap = event.getAttributes().getAttributeMap();
+    for (String statuscode : RpcSemanticConventionUtils.getAttributeKeysForGrpcStatusCode()) {
+      if (attributeValueMap.get(statuscode) != null) {
+        return Integer.parseInt(attributeValueMap.get(statuscode).getValue());
+      }
+    }
+    return -1;
+  }
+
+  public static String getGrpcStatusMsg(Event event) {
+    Map<String, AttributeValue> attributeValueMap = event.getAttributes().getAttributeMap();
```

Comment:
null check for event and attributes?

Reply:
it is already present when this method is called